node/localnode.go

```diff
@@ -27,9 +27,9 @@ import (
 )
 
 const (
-	protocolVersion              = 1
-	minCompatibleProtocolVersion = 0
-	maxCompatibleProtocolVersion = 10
+	protocolVersion              = 11
+	minCompatibleProtocolVersion = 11
+	maxCompatibleProtocolVersion = 12
```

Comment:
What about changing maxCompatibleProtocolVersion to 20 so that we have the flexibility to do multi-step upgrade, e.g. A -> B -> C, where A and B are compatible, B and C are compatible, but A and C are incompatible.

PS: we can also decrease or reset protocol version number in the future

Reply:
Done.